src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1472,7 +1473,7 @@ def maybe_free_model_hooks(self):
             hook.remove()
 
         # make sure the model is in the same state as before calling it
-        self.enable_model_cpu_offload()
+        self.enable_model_cpu_offload(device=getattr(self, "_offload_device", "cuda"))
```

Comment:
Personally I think it's a non-breaking change since we default to "cuda" for `device` in `enable_model_cpu_offload()`.